nrf_security/Kconfig

```diff
@@ -557,7 +557,7 @@ endif # MBEDTLS_CIPHER_MODE_OFB
 #
 config MBEDTLS_CIPHER_MODE_XTS
 	bool "AES-XTS - AES Xor-encrypt-xor with ciphertext stealing mode"
-	depends on OBERON_MBEDTLS_AES_C || VANILLA_MBEDTLS_AES_C
```

Comment:
I thought that we changed that already, a better solution will have been to rename the file xts_oberon.c to aes_oberon.c. This should solve the issues that we had. @frkv When we think that it's ok to rewrite history I can do that.

Reply:
Add it as a single commit and then we can rewrite history later.

Reply:
Added that